ui/account_details_page.go

```diff
@@ -149,49 +143,92 @@ func (pg *acctDetailsPage) accountBalanceLayout(gtx layout.Context, common *page
 
 	tMain, tSub := breakBalance((*pg.acctInfo).TotalBalance)
 	spendable := dcrutil.Amount((*pg.acctInfo).SpendableBalance).String()
+	immatureRewards := dcrutil.Amount((*pg.acctInfo).Balance.ImmatureReward).String()
+	imReMain, imReSub := breakBalance(immatureRewards)
+	lockedByTickets := dcrutil.Amount((*pg.acctInfo).Balance.LockedByTickets).String()
+	lockMain, lockSub := breakBalance(lockedByTickets)
+	votingAuthority := dcrutil.Amount((*pg.acctInfo).Balance.VotingAuthority).String()
+	voteMain, voteSub := breakBalance(votingAuthority)
+	immatureStakeGen := dcrutil.Amount((*pg.acctInfo).Balance.ImmatureStakeGeneration).String()
+	imStMain, imStSub := breakBalance(immatureStakeGen)
 	sMain, sSub := breakBalance(spendable)
 
 	return pg.pageSections(gtx, func(gtx C) D {
 		accountIcon := common.icons.accountIcon
 		if (*pg.acctInfo).Name == "imported" {
 			accountIcon = common.icons.importedAccountIcon
 		}
-		accountIcon.Scale = 0.8
+		accountIcon.Scale = 1
 
 		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
 			layout.Rigid(func(gtx C) D {
-				return layout.Flex{}.Layout(gtx,
+				return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
 					layout.Rigid(func(gtx C) D {
 						m := values.MarginPadding10
-						inset := layout.Inset{
+						return layout.Inset{
 							Right: m,
 							Top:   m,
-						}
-						return inset.Layout(gtx, func(gtx C) D {
+						}.Layout(gtx, func(gtx C) D {
 							return accountIcon.Layout(gtx)
 						})
 					}),
 					layout.Rigid(func(gtx C) D {
-						return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-							layout.Rigid(func(gtx C) D {
-								mainLabel := pg.theme.H4(tMain)
-								subLabel := pg.theme.Body1(tSub)
-								return acctBalLayout("Total Balance", mainLabel, subLabel)
-							}),
-							layout.Rigid(func(gtx C) D {
-								mainLabel := pg.theme.Body1(sMain)
-								subLabel := pg.theme.Caption(sSub)
-								inset := layout.Inset{
-									Top: values.MarginPadding5,
-								}
-								return inset.Layout(gtx, func(gtx C) D {
-									return acctBalLayout("Spendable", mainLabel, subLabel)
-								})
-							}),
-						)
+						mainLabel := pg.theme.H4(tMain)
+						subLabel := pg.theme.Body1(tSub)
+						return acctBalLayout("Total Balance", mainLabel, subLabel, true)
 					}),
 				)
 			}),
+			layout.Rigid(func(gtx C) D {
+				mainLabel := pg.theme.Body1(sMain)
+				subLabel := pg.theme.Caption(sSub)
+				inset := layout.Inset{
+					Top: values.MarginPadding5,
+				}
+				return inset.Layout(gtx, func(gtx C) D {
+					return acctBalLayout("Spendable", mainLabel, subLabel, false)
+				})
+			}),
```

Comment:
This layout looks repeated. It can be made a function or method and passed the data that needs to be laid out.